src/components/Basic/ConnectButton.tsx

```diff
@@ -319,16 +319,18 @@ function ConnectButton() {
                   <img src={arrowRightImg} alt="arrow" />
                 </div>
 
-                <div
-                  className="flex align-center just-between metamask-connect-btn"
-                  onClick={() => onConnect(ConnectorNames.WalletConnect)}
-                >
-                  <div className="flex align-center">
-                    <img src={walletconnectImg} alt="coinbase wallet" />
-                    <span>WalletConnect</span>
+                {process.env.REACT_APP_CHAIN_ID === '56' && (
```

Comment:
Would be nice to have a short util like `isDev`

Reply:
We do in `develop`: 'isOnTestnet`.
This is just a hotfix so I wanted to write as less code as possible. I've opened a draft PR to introduce this fix too, there it does use the config appropriately: https://github.com/VenusProtocol/venus-protocol-interface/pull/269